src/Console/PurgeCommand.php

```diff
@@ -32,33 +33,44 @@ class PurgeCommand extends Command
      */
     public function handle()
     {
-        $expired = $this->option('hours')
+        $revoked = $this->option('revoked') || ! $this->option('expired');
+        $expired = $this->option('expired') || ! $this->option('revoked')
             ? Carbon::now()->subHours($this->option('hours'))
-            : Carbon::now()->subDays(7);
+            : false;
 
-        if (($this->option('revoked') && $this->option('expired')) ||
-            (! $this->option('revoked') && ! $this->option('expired'))) {
-            Passport::token()->where('revoked', 1)->orWhereDate('expires_at', '<', $expired)->delete();
-            Passport::authCode()->where('revoked', 1)->orWhereDate('expires_at', '<', $expired)->delete();
-            Passport::refreshToken()->where('revoked', 1)->orWhereDate('expires_at', '<', $expired)->delete();
+        Passport::token()->where(function (Builder $query) use ($revoked, $expired) {
+            $query->when($revoked, function ($query) {
```

Comment:
What do you think about this style?

```
$query->when($revoked, fn (Builder $query) => $query->orWhere('revoked', true));
```

Reply:
I prefer that, but the `when` closure doesn't return the `query` instance [according to docs](https://laravel.com/docs/11.x/queries#conditional-clauses), using arrow functions it does!

Reply:
Got it, but I think we should add type for $query ?

Reply:
Changed to arrow function as its seems ok according to the doc type, thanks:

https://github.com/laravel/framework/blob/68e88bbcee1e9b0b02ca267916ee25f45ab3935c/src/Illuminate/Conditionable/Traits/Conditionable.php#L17